src/plugins/alwaysAnimate/index.ts

```diff
@@ -51,7 +51,7 @@ export default definePlugin({
             // Guild Banner
             find: ".animatedBannerHoverLayer,onMouseEnter:",
             replacement: {
-                match: /(?<=guildBanner:\i,animate:)\i(?=}\))/,
+                match: /(?<=guildBanner:\i,animate:)\i(?=,isRefreshEnabled:)/,
```

Comment:
```suggestion
                match: /(?<=guildBanner:\i,animate:)\i/,
```

Lets just nuke it to not depend on visual refresh stuff

Reply:
if we remove that, and we should, it should be anchored to something that isn't just an identifier so it doesn't match a destructuring assignment and error

Reply:
```suggestion
                match: /(?<=guildBanner:\i,animate:)\i(?=.{0,100}controller)/,
```
i feel like this is the best i can do to not rely on a visualReferesh prop